### PR TITLE
Black formatter

### PR DESCRIPTION
**Pull request by TheMythologist**
Ran `black -l 132` amongst other miscellaneous formatting changes

### CODE REVIEW

**Comment by doomedraven** [comment]
thank you, i have found one interesting "issue", that `except ImportError` was replaced with `except Exception`, which tool did you use do recursive changes? apart of that everything looks good, just be careful with `re2` as i specified in another PR

**Comment by TheMythologist** [comment]
Oh, I intentionally changed that error to be more specific. I simply used vscode regex to do recursive changes (replacing all instances of `except:` with `except Exception:`.

Yup I will take note of the re2 python import issue, thanks for the merge :)

**Comment by doomedraven** [comment]
em not make sense replace specific error with generic one, at least i don't see, let me know if i missing something

thanks for cleanup

**Comment by TheMythologist** [comment]
oh I replaced `except Exception` with `except ImportError`, not the other way around 😅

**Comment by doomedraven** [comment]
np, in only one case that i spot `except ImportError` was replaced with `except Exception` on yara importing

File: lib/cuckoo/common/office/rtfdump.py [hunk 1]
```diff
@@ -47,17 +47,21 @@
...
 
+try:
+    import re2 as re
+except ImportError:
+    import re
+
 try:
     import yara
-except Exception:
+except ImportError:
```
Comment (TheMythologist) [inline lib/cuckoo/common/office/rtfdump.py:64]: @doomedraven I think you're referring to here where I replaced `except Exception` with `except ImportError`

Comment (doomedraven) [reply lib/cuckoo/common/office/rtfdump.py:64]: em ok than the cherry-picking should did a mess, thanks

**Comment by doomedraven** [comment]
ok problem started

```
Traceback (most recent call last):
  File "manage.py", line 15, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python3.8/dist-packages/django/core/management/__init__.py", line 419, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.8/dist-packages/django/core/management/__init__.py", line 363, in execute
    settings.INSTALLED_APPS
  File "/usr/local/lib/python3.8/dist-packages/django/conf/__init__.py", line 82, in __getattr__
    self._setup(name)
  File "/usr/local/lib/python3.8/dist-packages/django/conf/__init__.py", line 69, in _setup
    self._wrapped = Settings(settings_module)
  File "/usr/local/lib/python3.8/dist-packages/django/conf/__init__.py", line 170, in __init__
    mod = importlib.import_module(self.SETTINGS_MODULE)
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/CAPEv2/web/web/settings.py", line 26, in <module>
    from lib.cuckoo.core.startup import init_rooter, init_routing
  File "/opt/CAPEv2/web/../lib/cuckoo/core/startup.py", line 37, in <module>
    from lib.cuckoo.core.plugins import import_plugin, import_package, list_plugins
  File "/opt/CAPEv2/web/../lib/cuckoo/core/plugins.py", line 16, in <module>
    from lib.cuckoo.common.abstracts import Auxiliary, Machinery, LibVirtMachinery, Processing
  File "/opt/CAPEv2/web/../lib/cuckoo/common/abstracts.py", line 34, in <module>
    from lib.cuckoo.common.url_validate import url as url_validator
  File "/opt/CAPEv2/web/../lib/cuckoo/common/url_validate.py", line 19, in <module>
    regex = re.compile(  # noqa: W605
  File "compile.pxi", line 6, in re2.compile
  File "compile.pxi", line 93, in re2._compile
re.error: invalid escape sequence: \u
```

**Comment by doomedraven** [comment]
so i guess everything where `u` where added to strings and re2 added is broken now

**Comment by doomedraven** [comment]
i have reverted everything where re2 was added

**Comment by doomedraven** [comment]
also removed everything where `u` was added, idk what you have used, but in future please use `black -t py38 -l 132` + `DO TESTING!`

**Comment by TheMythologist** [comment]
None of my commits added `u` to any string.

But yes it seems that re2 and re have differences in syntax

**Comment by TheMythologist** [comment]
Anyways isolated the error to be re2 having different syntax for unicode escape characters

**Comment by TheMythologist** [comment]
And also unicode formatting is a deprecated python2 thing, so it's fine to remove it in front of all strings since python3 is exclusively used for CAPE

**Comment by doomedraven** [comment]
Yes but with this merge i got tons of unicode strings doesnt matter now we
learned new lesson what to care about

**Comment by kevoreilly** [comment]
The real point is the testing.

**Pull request merged**